docs/readmes/lte/extended_5G_sa_features.md

```diff
@@ -0,0 +1,173 @@
+On occurrence of a failure, AGW will retrieve the UE data from the persistent storage, a partial service interruption may occur. However, once the AGW has been respawned then registered users can access the service without having a fresh registration.
+
+The MME will sync UE registration and session management data into persistent storage (Redis Database) for pre-defined trigger points. Incase of any MME failure/restart, MME will fetch UE specific data from Redis Database.
+
+![Stateless Feature](assets/lte/Stateless_feature.png?raw=true "Stateless Feature")
+
+## GTP extension header support
+
+The Extension Header Length field specifies the length of the particular Extension header in 4 octet units. The Next Extension Header Type field specifies the type of any Extension Header that may follow a particular Extension Header. If no such Header follows, then the value of the
+Next Extension Header Type shall be 0.
+
+```text
+	 0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+     | Ext-Hdr Length|                                               |
+     +-+-+-+-+-+-+-+-+                                               |
+     |                  Extension Header Content                     |
+     .                                                               .
+     .                                               +-+-+-+-+-+-+-+-+
+     |                                               |  Next-Ext-Hdr |
+     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+```
+
+In 5G SA, each flow is forwarded based on the appropriate QoS rules. QoS rules are configured by SMF as QoS profiles to UP components and these components perform QoS controls to PDUs based on rules. In downlink, a pipelineD pushes QFI into an extension header, and transmits the PDU to RAN.  In uplink, each UE obtains the QoS rule from SMF, and transmits PDUs with QFI containing the QoS rules to the RAN.
+
+## Additional AGW Configurations
+
+### Network initiated Session Modification
+
+Stub CLI used to add the QoS policy in Network initiated session modification.
+
+```text
+session_manager_cli.py policy_rar IMSI001019976545660 IMSI001019976545660-112383 INTERNET ADD 10 6 "UL,6,192.168.128.12,0,192.168.200.1,12345;DL,1,8.8.8.8,0,192.168.128.12,0" "1000000,2000000,1000000,2000000,15,1,0"
+```
+
+Stub CLI used to delete the QoS policy in Network initiated session modification.
+
+```text
+session_manager_cli.py policy_rar IMSI001019976545660 IMSI001019976545660-112383 INTERNET DEL 10 6 "UL,6,192.168.128.12,0,192.168.200.1,12345;DL,1,8.8.8.8,0,192.168.128.12,0" "1000000,2000000,1000000,2000000,15,1,0"
+```
+
+Here is the details of the [Stub CLI command](assets/lte/Session_modification_stub_cli_commands "Stub CLI Command") for add or delete the QoS policy in Network initiated session modification.

```

Comment:
- `Here are the details`
- The link seems broken.

Reply:
![image](https://user-images.githubusercontent.com/89978170/186345605-12313606-ea9a-4fd7-a593-4faec8b98ff7.png)
On clicking the "Stub CLI Command", a document (`docs/readmes/assets/lte/Session_modification_stub_cli_commands`) will be downloaded.

Reply:
On clicking the "Stub CLI Command", a document (`docs/readmes/assets/lte/Session_modification_stub_cli_commands`) will be downloaded.

Reply:
You are right. The "tooltip function" for markdown links is not mentioned within the [docusaurus documentation](https://docusaurus.io/docs/markdown-features/assets), such that I was unaware it is indeed supported.